frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -423,10 +424,13 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             },
         ],
     })),
-    reducers(({ props }) => ({
+    reducers(({ props, values }) => ({
         orderBy: [
-            'start_time' as RecordingsQuery['order'],
-            { persist: true },
+            (values.featureFlags[FEATURE_FLAGS.REPLAY_DEFAULT_SORT_ORDER_EXPERIMENT] === 'control' ||
```

Comment:
Can I assume this feature flag will be inited before this code runs?

Reply:
you can connect `featureFlags` as a value
and then have a selector that exposes the flag

that will change when the values change
and answering the delayed assignment question (I think) that lets you return `null` or `"not-yet"` or whatever until you see that flags have values or can be sure that decide response has been received